siriuspy/siriuspy/injctrl/main.py

```diff
@@ -1050,6 +1096,29 @@ def _callback_update_pu_refvolt(self, pvname, value, **kws):
         index = self._pu_names.index(devname)
         self._pu_refvolt[index] = value
 
+    def _callback_is_injecting(self, value, **kws):
+        if value is None:
+            return
+
+        # check if any InjSI PU is pulsing
+        is_injecting = False
+        for pvs in self._pvs_injsys.values():
+            if not all([pvo.connected for pvo in pvs]):
+                break
```

Comment:
I think if they are not connected we must assume the worst scenario.

Reply:
done!